### PR TITLE
fixing react lint warnings

### PR DESCRIPTION
**Pull request by atomantic**

### CODE REVIEW

File: src/context/HicetnuncContext.js [hunk 1]
```diff
@@ -22,8 +22,6 @@ export default class HicetnuncContextProvider extends Component {
 
       address: '',
 
-      op: undefined,
```
Comment (atomantic) [inline src/context/HicetnuncContext.js:25]: this is duplicated. I find that alphabetizing object keys adds a nice way to make sure things are doubled and also helps code collaborators have fewer merge conflicts when adding keys :)

File: src/pages/about.js [hunk 2]
```diff
@@ -22,23 +22,23 @@ export default class About extends Component {
...
+    //   listStyle: 'none',
+    //   right: '0',
+    //   top: '0',
+    //   marginTop: '15%',
+    //   marginRight: '25px',
+    //   fontFamiliy: 'roboto',
+    //   textAlign: 'right',
+    //   fontSize: '40px',
+    //   animation: 'fadeMe 1.2s',
+    // }
```
Comment (atomantic) [inline src/pages/about.js:41]: these are unused...

File: src/pages/display.js [hunk 3]
```diff
@@ -216,6 +217,7 @@ export default class Display extends Component {
                       </span>
                       <span onClick={this.collection}>
                         <a
+                          href="#collection"
```
Comment (atomantic) [inline src/pages/display.js:220]: nice to be able to link to these anyway :)

File: src/pages/feed.js [hunk 4]
```diff
@@ -42,17 +42,15 @@ export default class Feed extends Component {
...
             e.token_id !== 1131 &&
             e.token_id !== 1417 &&
             e.token_id !== 1418 &&
             e.token_id !== 1419 &&
             e.token_id !== 641 &&
             e.token_id !== 1547
-          ) {
-            return e
-          }
+          )
```
Comment (atomantic) [inline src/pages/feed.js:53]: same thing but always returns `true` or `false` instead of `true` or `undefined`

**Comment by andrevenancio** [comment]
There's a few conflicts now. I've been doing a bit of cleanup myself but there's a lot of places that still need work. Its probably best to approach it with smaller PR's almost file by file.

**Comment by andrevenancio** [comment]
Some conflicts here and some design changes being introduced since this PR was created. Here's a list from the top of my head so you could amend this PR accordingly.

- The idea is to go through each file and remove the unused properties. instead of commenting them out.
- there's a clear distinction between <a> <Link> and <button> an <a> tag opens an external page. a <Link> is used for route changes (using react-router-dom) and finally a <button> is used for onClick events. all of this is handled by src/components/button/index.js so you should use the <Button/> component.

If possible keep PR's as small as possible so its easier to avoid conflicts and quickly merge them. There's a lot of warning everywhere right now, it annoys me a bit, but we need to fix them page/component at a time.

**Comment by atomantic** [comment]
I'll break this up into new pull requests :)

**Pull request closed without merging**